aten/src/ATen/CachedTensorUtils.cpp

```diff
@@ -0,0 +1,49 @@
+#include <ATen/ATen.h>
+
+#include <ATen/CachedTensorUtils.h>
+
+
+namespace at {
+namespace caching {
```

Comment:
This is not doing any caching so it is very weird to use such a namespace for it?

Reply:
what would you suggest ?  I had something more cudagraph specific originally.